### DISCUSSION TITLE
Mettre à jour les graveurs sur GRBL et les messages codes interne en multi langues ?

### DISCUSSION THREAD

**Discussion by DESIRIUS38** [arkypita/LaserGRBL#2196] [Ideas]
Beaucoup dispose de l' ASTOMSTACK S10 Pro (Les trois modèles X7 Pro, A10 Pro et S10 Pro sont identiques, seule la couleur du cadre est différente.) La configuration n'existe pas dans matériel database ? je suis désolé si je ne suis pas très alaise j'ai 60 ans et je débute laser et cnc merci à vous il met difficile de configurer mon laser qui semble pas découper comme le fabriquant l'indique..

**Comment by Tribo-2dot0** [comment]
Ajuste o foco o melhor possível e faça uma grade de teste para corte.
https://kpc29laser.fr/MGL/MGL.html
